xcodeproj/internal/updater.template.sh

```diff
@@ -42,7 +42,10 @@ for i in "${!updated_xcodeprojs[@]}"; do
     exit 1
   fi
 
-  diff=$(git diff "$xcodeproj")
+
+  # Exclude swiftc diff for now since this can be different when built with
+  # different Xcode versions.
+  diff=$(git diff -- ":(exclude)*/swiftc" "$xcodeproj")
```

Comment:
Ideally we don't have binaries checked in. So I'm thinking that we should change the `include_spec` to `for_fixture` and extend its behavior to create a relative symlink for `swiftc`. That way the fixtures still work fully when opened. Because this will only work if your output base is in-repo, I don't think it should be the default behavior.

Thoughts?

Reply:
That makes sense. But the path of `swiftc` depends on the host arch, so I'm not sure how to make a relative symlink works. Does it need to be a valid binary?

Reply:
For the fixtures, I'm making it an empty file for now.

Reply:
The fixtures transition the host architecture.

Reply:
And yes, fixtures need to be runnable.

Reply:
After resolving this (either with the symlink or updating the docs), we should be good to merge.

Reply:
Added a comment in the CONTRIBUTING.md file.